mantidproject/mantid
# Indirect Calculate and Apply corrections fixes

Fixes [#11480](http://trac.mantidproject.org/mantid/ticket/11480).

To test, check that:
- When Flat Plate is the geometry, the can back thickness is set to whatever the can front thickness is when it is changed
- When Flat Plate is the geometry, the can thickness options are disabled when Use Can is unticked
- When Cylinder is the geometry, the can radius options are disabled when Use Can is unticked
- There is an option for annular geometry and an entry on the documentation (this could just be code reviewed if not on Windows)
- The output of CalcCorr has a vertical axis of ElasticQ and plot options work as expected
- The naming of the result matches the geometry (`cyl`, `flt`, `ann` in workspace name)
- ApplyCorr will only browse for corrections of the correct geometry (based on the naming of the workspace)
